src/SablierV2NFTDescriptor.sol

```diff
@@ -248,6 +249,7 @@ contract SablierV2NFTDescriptor is ISablierV2NFTDescriptor {
 
     /// @notice Generates a string with the NFT's JSON metadata description, which provides a high-level overview.
     function generateDescription(
+        bool isTransferable,
```

Comment:
Could we please define this parameter last? After `streamId`

Reply:
after `streamId` doesn't mean last:

https://github.com/sablier-labs/v2-core/blob/e8c604733c40cb70accf0ec410cca0d785b6cbd7/src/SablierV2NFTDescriptor.sol#L250-L258

Reply:
Let's move `streamId` after `assetAddress`, and `isTransferable` after `assetAddress`

Reply:
ok 😅

Reply:
Out of curiosity, why didn't you go with alphabetical order in this case?

Reply:
@smol-ninja cuz there is a natural hierarchy: contract variables first, followed by stream data.

In general: alphabetical is the default unless there's another more specific rule that makes sense in a particular case.

But this is more art than science; if you prefer, I would be open to reordering the params alphabetically.

Reply:
Let's be artists. Gtg.